workshop/content/exercises/deploy-sample-app.adoc

```diff
@@ -10,20 +10,28 @@ We can verify the name of the project with:
 oc project -q
 ----
 
-You will use the link:https://github.com/sclorg/nodejs-ex[nodejs-ex] sample application during this tutorial, which is a simple nodejs application.
+You will use the link:https://github.com/sclorg/nodejs-ex[nodejs-ex] sample application during this tutorial,
+which is a simple Node.js application.
 
-Create the Kubernetes objects for deploying the nodejs-ex app on OpenShift by running
-the command below. The deployment will not complete since there are no container images
-built for the nodejs-ex application yet. That you will do in the following sections
-through a CI/CD pipeline.
+To prepare for nodejs-ex's eventual deployment, you will create Kubernetes objects that
+are supplementary to the application, such as a route (i.e. url). The deployment will not
+complete since there are no container images built for the nodejs-ex application yet.
+That you will do in the following sections through a CI/CD pipeline.
+
+Create the supplementary Kubernetes objects by running the command below:
 
 [source,bash,role=execute-1]
 ----
 oc create -f sampleapp/sampleapp.yaml
 ----
 
 To verify the resources needed to support nodejs-ex have been created, you can head
-out to the OpenShift web console by clicking on the **Console** tab next to the
+out to the OpenShift web console. A warning that there will be an error message first
```

Comment:
Added warning about developer console starting on page with an error message.

Reply:
Good, but slightly awkward to read.

Perhaps an admonition like you did elsewhere (I forget exactly where, but the Note call out). And maybe slightly rephrase to "An error message may appear when you first visit the console due to the fact that the user does not have permissions to the default project."

Reply:
I touched up the language and bolded a **Note** for it. This is technically not an error as the user is set up to not be able to see other projects, so I think calling it out via a note rather than a warning should work and will hopefully not scare the end user as much.